stock-managment/src/main/java/org/dhis2/android/rtsm/ui/managestock/ManageStockViewModel.kt

```diff
@@ -95,6 +96,12 @@ class ManageStockViewModel @Inject constructor(
     private val _stockItems: MutableLiveData<PagedList<StockItem>> =
         MutableLiveData<PagedList<StockItem>>()
 
+    private val _canReview = MutableStateFlow(false)
+    val canReview: StateFlow<Boolean> = _canReview
+
+    private val _isEditing = MutableStateFlow(false)
+    val isEditing: StateFlow<Boolean> = _isEditing
+
```

Comment:
why don't you have a simpler stateflow to show the status of the button? like 
`enum buttonStatus {hidden, disabled, enabled}`
and the check the status in the view, this allows you to move all the loging that you have in the view to the viewmodel